components/profile_picture.jsx

```diff
@@ -26,6 +26,7 @@ export default class ProfilePicture extends React.PureComponent {
         isBusy: PropTypes.bool,
         isRHS: PropTypes.bool,
         hasMention: PropTypes.bool,
+        helperClass: PropTypes.string,
```

Comment:
Maybe we could call this `wrapperClass` since that has a bit more meaning than `helperClass`?

Reply:
Yep, I'm fine with that, since it's specifically part of the wrapper.